.gitignore

```diff
@@ -0,0 +1,14 @@
+# OS X
+.DS_Store
+
+# Xcode
+build/
+/*.xcodeproj
```

Comment:
Despite this package won't contain any .xcodeproj files - this specific ignore is generally more harmful than helpful. Please remove this line.